plugins/nvm/_nvm

```diff
@@ -1,7 +1,9 @@
 #compdef nvm
 #autoload
 
-[[ -s ~/.nvm/nvm.sh ]] || return 0
+if ![ -f $HOME/.nvm/nvm.sh ] && ![ $commands[brew] -a -f `brew --prefix`/etc/profile.d/autojump.sh ]; then
```

Comment:
Whoops. Fixed. I'll see if I can change to the double bracket syntax. As you can guess, I copied this from the [autojump](https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/autojump/autojump.plugin.zsh) file.